api/ChattO/API/Controllers/OrganizationController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using API.DTOs.Requests.Orqanization;
+using API.DTOs.Responses.Organization;
+using Application.Organizations.Commands;
+using Application.Organizations.Queries;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+public class OrganizationController : BaseController
+{
+    [HttpPost]
+    [AllowAnonymous]
```

Comment:
Why we need it? Controller have not be marked as [Authorize]

Reply:
AllowAnonymous is needed because the first user (organization) does not have an account, all other endpoints will be accessible by authenticated users (super admin)